src/script/properties/PropertiesService.ts

```diff
@@ -17,24 +17,17 @@
  *
  */
 
-import {BackendClient} from '../service/BackendClient';
+import {APIClient} from '@wireapp/api-client';
 
 class PropertiesService {
-  private readonly backendClient: BackendClient;
-
-  // tslint:disable-next-line:typedef
-  static get CONFIG() {
-    return {
-      URL_PROPERTIES: '/properties',
-    };
-  }
+  private readonly apiClient: APIClient;
 
   /**
    * Construct a new Properties Service.
-   * @param {BackendClient} backendClient - Client for the API calls
+   * @param {APIClient} apiClient - Client for the API calls
```

Comment:
```suggestion
   * @param apiClient Client for the API calls
```

Reply:
Let's remove the whole comment block since the types are self-explaining.